src/GitHub.VisualStudio/UI/Views/Controls/PullRequestListItem.xaml

```diff
@@ -57,7 +57,7 @@
             TextTrimming="CharacterEllipsis"
             Margin="0,-3,5,0"
             Content="{Binding Title}"
-            ToolTip="{Binding Title}"
+            ToolTip="View Pull Request details"
```

Comment:
We normally put strings like this in the resources file for when we start adding in translations.